packages/lesmis/src/views/DashboardView.js

```diff
@@ -483,14 +483,14 @@ const VitalsView = ({ vitals }) => {
 export const DashboardView = () => {
   const { entityCode } = useUrlParams();
   const { data: entityData } = useEntityData(entityCode);
-  const vitals = useVitalsData(entityCode);
-
   const tabOptions = makeTabOptions(entityData?.type);
+  const [params, setParams] = useUrlSearchParams();
 
-  const [selectedTab, setSelectedTab] = useState(tabOptions[0].value);
+  const vitals = useVitalsData(entityCode);
+  const selectedTab = params.dashboardTab || tabOptions[0].value;
 
   const handleChangeTab = event => {
-    setSelectedTab(event.target.value);
+    setParams({ dashboardTab: event.target.value, dashboard: null, year: null });
```

Comment:
@reviewer I would be interested to hear your thoughts on `dashboard`and `dashboardTab` as the names of the params.

Reply:
I prefer `dashboard` and `subDashboard`. In fact, when I saw `dashboardTab` I thought it was going to mean these guys
![image](https://user-images.githubusercontent.com/1274422/119077036-b2ca9f80-ba47-11eb-9edc-8d5e10f17fbc.png)

rather than these guys
![image](https://user-images.githubusercontent.com/1274422/119077045-b6f6bd00-ba47-11eb-856c-f32881ff9ba6.png)

Following up on that, though outside the scope of this PR, because the second one is a drop down rather than tabs, can you rename things that refer to them as tabs? E.g. the `makeTabOptions` within this file, which actually makes drop down options.